DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    cutout = ExposureF(mem)
</pre><p>Could you add a note about the meaning of ExposureF()?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>It made the most sense to me to put this in the header for the make_image_cutout function. Added!</p>